mmv1/templates/terraform/examples/firebase_app_hosting_backend_github.tf.tmpl

```diff
@@ -0,0 +1,70 @@
+resource "google_firebase_app_hosting_backend" "example" {
```

Comment:
Since these are both going to be shown in the docs as examples, we should be consistent with the ordering ie. dependencies-first or dependencies-last. I think dependencies-last is more common, but either should be fine.